var/spack/repos/builtin/packages/casacore/package.py

```diff
@@ -12,26 +12,39 @@ class Casacore(CMakePackage):
     homepage = "https://github.com/casacore/casacore"
     url      = "https://github.com/casacore/casacore/archive/v2.4.1.tar.gz"
 
```

Comment:
Would you like to add yourself as a maintainer? Just add:
```python
maintainers = ['mpokorny']
```
That way, if someone has trouble building the `casacore` package, or wants to submit a PR to update it, we can ping you to take a look.

Reply:
Sure, I'll do that